omod/src/main/webapp/resources/js/custom/app.js

```diff
@@ -220,6 +221,35 @@ muzimaCoreModule.factory('FileUploadService', function ($http) {
     };
 });
 
+muzimaCoreModule.factory('HtmlFormEntryService', function ($http) {
+
+    var moduleState = function () {
+
+        return $http.get('htmlFormEntrymoduleStatus.form');
```

Comment:
For consistency purposes consider naming this using camelCase. m Should be capital. Likewise htmlformentry.form should be in camelCase